mgmtsystem_nonconformity/models/mgmtsystem_nonconformity.py

```diff
@@ -218,27 +227,71 @@ def create(self, vals):
 
     @api.multi
     def message_auto_subscribe(self, updated_fields, values=None):
-        """Add the responsible, manager and OpenChatter follow list."""
+        """
+        Add the responsible, manager and author to the OpenChatter follow list
+        """
         self.ensure_one()
         user_ids = [
             self.responsible_user_id.id,
             self.manager_user_id.id,
             self.author_user_id.id,
         ]
         self.message_subscribe_users(user_ids=user_ids, subtype_ids=None)
```

Comment:
OK, I suggest that you comment out that code and add a 
`# TODO` comment. We'll go ahead with the PR and in a later PR add the messaging logic.